Rules.md

```diff
@@ -186,58 +182,51 @@ When a move proceeds into an enemy region, a battle is initiated with all moved
 A battle has one or more rounds. Every round has following parts:
 
 1. **Artillery strike:**
-    * Every healthy attacking artillery kills 1 unit (*)
+    * Every attacking artillery kills 1 unit (*)
 1. **Rolling dice:**
-    * Attacker gets a dice for every healthy attacking **infantry unit**
-    * Defender gets a dice for every healthy defending **unit** (infantry + artillery)
+    * Attacker gets a dice for every attacking **infantry unit**
+    * Defender gets a dice for every defending **unit** (infantry + artillery)
     * Every attacker roll of:
-        * **3-4:** Wounds an enemy unit (*)
-        * **5-6:** Kills an enemy unit (*)
+        * **3-4:** Wounds an enemy unit
+        * **5-6:** Kills an enemy unit
     * Every defender roll of:
-        * **1-4:** Wounds an enemy unit (*)
-        * **5-6:** Kills an enemy unit (*)
+        * **1-4:** Wounds an enemy unit
+        * **5-6:** Kills an enemy unit
 
-    Kills are resolved before wounds.
-1. **Attacker decides to retreat or has no healthy units left:** All units move back to the region
-   where to move started.
+   Wound triggers a retreat for the wounded unit(s).
+   Kills are resolved before wounds. Infantries are targeted before artilleries.
+1. **Attacker retreat:** Wounded and other selected units move back to the region where the move
+   started.
 
-   Edge case: If the attacker has no healthy units left and the defender has no units left, the
-   attacker has to retreat. Any command tokens on the attacked region are returned to the defender's
-   supply.
-1. **Defender decides to retreat or has no healthy units left:** All units (including wounded units)
-   move to a single adjacent land region that is either unoccupied or owned by the defender.
-   Exception: units can't retreat to the region where the attacker's move started.
+   If no attacking units are left, the battle is instantly over and any wounded defending
+   units will not retreat.
+1. **Defender retreat:** Wounded and other selected units move to a single adjacent land region that
+   is either owned by the defender or unoccupied, but not the region where the attacker's move
+   started. Retreating units in subsequent battle rounds need to retreat to this same area.
```

Comment:
? no I meant that the defender can choose each battle round where his wounded units retreat to